iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/project/tara/oozie_app/workflow.xml

```diff
@@ -35,7 +35,7 @@
             <description>output document to project</description>
         </property>
         <property>
-            <name>taraSparkSqlShufflePartitions</name>
+            <name>sparkSqlShufflePartitions</name>
```

Comment:
Thanks for renaming. I just added this property to the `config-default.xml` template:

https://git.icm.edu.pl/openaire/iis-deployment/-/commit/e37db9933a5dc99970bfe465decf440bd862ebf3#2a1f4f26f9126891547a6e49fa344ea3698ee89d

it could be used also in other places whenever needed.